pymdoccbor/mso/issuer.py

```diff
@@ -208,19 +204,27 @@ def sign(
             "deviceKeyInfo": {
                 "deviceKey": device_key,
             },
-            "digestAlgorithm": alg_map.get(self.alg),
+            "digestAlgorithm": alg_map.get(self.alg)
         }
-
+        if self.status_list is not None:
+            payload.update({"status": self.status_list})
         if self.revocation is not None:
             payload.update({"status": self.revocation})
 
         if self.cert_path:
-            # Load the DER certificate file
+            # Try to load the certificate file
             with open(self.cert_path, "rb") as file:
                 certificate = file.read()
-
-            cert = x509.load_der_x509_certificate(certificate)
-
+            try:
+                cert = x509.load_pem_x509_certificate(certificate)
+            except Exception as e:
+                logger.error(f"Certificate at {self.cert_path} could not be loaded as PEM, trying DER")
+                try:
+                    cert = x509.load_der_x509_certificate(certificate)
+                except Exception as e:
+                    _err_msg = f"Certificate at {self.cert_path} could not be loaded as DER"
+                    logger.critical(_err_msg)
+                    raise Exception(_err_msg)
```

Comment:
this needs `from cryptography.x509 import Certificate`

this code prevents a useless and costly try/except using a conditional 

```suggestion
            _parsed_cert: Union[Certificate, None] = None
            try:
                _parsed_cert = x509.load_pem_x509_certificate(certificate)
            except Exception as e:
                logger.error(f"Certificate at {self.cert_path} could not be loaded as PEM, trying DER")
            
            if not _parsed_cert:
                try:
                    _parsed_cert = x509.load_der_x509_certificate(certificate)
                except Exception as e:
                    _err_msg = f"Certificate at {self.cert_path} could not be loaded as DER"
                    logger.critical(_err_msg)
                    raise Exception(_err_msg)
             
             _parsed_cert = cert
```

Reply:
@oligatorr do you think to take also this one?